core/common/src/main/java/alluxio/wire/LogInfo.java

```diff
@@ -70,4 +70,9 @@ public String getMessage() {
   public void setMessage(String message) {
     mMessage = message;
   }
+
+  @Override
+  public String toString() {
+    return mLogName + ":" + mLevel + "." + (mMessage != null ? " message: " + mMessage : "");
```

Comment:
what about in the same format as in logs/*.log?

{mLevel} {mLogName} - {mMessage}

like `2016-12-22 12:34:11,573 INFO  server.Server (Server.java:doStart) - jetty-9.2.z-SNAPSHOT`